test/setup.js

```diff
@@ -1,13 +1,22 @@
 import Stdlib from '../src/models/stdlib/Stdlib'
+import ZosPackageFile from '../src/models/files/ZosPackageFile'
+import ZosNetworkFile from '../src/models/files/ZosNetworkFile'
 import StdlibInstaller from '../src/models/stdlib/StdlibInstaller'
 
+useTestZosPackageFile()
 doNotInstallStdlib()
 
 require('chai')
   .use(require('chai-as-promised'))
   .use(require('zos-lib').assertions)
   .should()
 
+function useTestZosPackageFile() {
+  ZosPackageFile.prototype.write = () => {}
+  ZosNetworkFile.prototype.write = () => {}
+  ZosPackageFile.prototype.ifMissingThrow = () => {}
```

Comment:
Why is this throw removed? I'd rather modify the classed being tested as little as possible while testing.

Reply:
Agree. I'm removing this line and forcing the init script instead when needed